packages/e2e-selenium/util/close-welcome-guide.js

```diff
@@ -0,0 +1,20 @@
+const { login } = require( './login' );
+const { createNewPost } = require( './create-new-post' );
+
+const closeWelcomeGuide = async () => {
```

Comment:
AFAIK, selenium doesn't have an `evaluate` in Puppeteer. Because of that, we cannot write code like below in `create-new-post` in `test-utils`.

```js
const isWelcomeGuideActive = await page.evaluate( () =>
	wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
);

if ( showWelcomeGuide !== isWelcomeGuideActive ) {
	await page.evaluate( () =>
		wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' )
	);

	await page.reload();
}
```

All we can do is open up a new page in the beginning and close it programmatically by sending ESC key.

That's what it does: closing welcome guide for the first time.

Note. I tried to mimic the code above with `driver.executeJavaScript` but [failed](https://automate.browserstack.com/builds/011fae724222641159b4fa92e5b2dec6b9954965/sessions/dda1fe74f5044501c8ab5c2300cf608c8e9de798?auth_token=86bced6361416f8fb897eba894897748d31d042a6eab74067ecff6ecece60625). Because it tries to find an element right after reloading started.

That's why `createNewPost` and `closeWelcomeGuide` are separate functions.

Reply:
Perhaps I wasn't clear enough. My issue is with the function being called `closeWelcomeGuide`, when what it is doing is: 
1. Login;
2. create new post;
3. close welcome guide.

It would be better to give this function a name that reflects all three of the things it does, instead of just the last one.

Reply:
How about fixing that in this way?

* Remove `login()` and `createNewPost()`.
* Call them in `beforeAll()`

It's clearer than simply calling `closeWelcomeGuide()` inside the `beforeAll()`.

Reply:
Sure, that would work!